cmd/lifecycle/analyzer.go

```diff
@@ -214,7 +215,8 @@ func (a *analyzeCmd) registryImages() []string {
 		registryImages = append(registryImages, a.platform06.cacheImageTag)
 	}
 	if !a.useDaemon {
-		registryImages = append(registryImages, a.analyzeArgs.imageName)
+		registryImages = append(registryImages, append([]string{a.imageName}, a.additionalTags...)...)
+		registryImages = append(registryImages, a.runImageRef, a.previousImage)
```

Comment:
Is it possible to combine this into one big `append`, so that it's more readable?

Reply:
Fixed in 942ee2c